packages/release/prepare-local-fs.service

```
@@ -10,8 +10,9 @@ RefuseManualStop=true
 [Service]
 Type=oneshot
 
+EnvironmentFile=/usr/lib/data-partition-filesystem
```

nit: should use `/usr/share` for non-executable content, and this could have a more generic name to entice us to extend it in the future with other feature flag env variables:
```suggestion
EnvironmentFile=/usr/share/bottlerocket/image-features.env
```

---

I debated back and forth on `/usr/lib/` and `/usr/share` and apparently debated myself into the wrong place :) I'll fix the path and I really like the more generic option for these variables. Thanks for the suggestion!